.github/workflows/test.yml

```diff
@@ -34,33 +38,33 @@ jobs:
     - name: Test with pytest
       run: |
         python -m pytest --ignore-glob="*nft_upload_test.py" test --cov=samila --cov-report=term
-      if: matrix.python-version != 3.5
+      if: matrix.python-version != env.TEST_PYTHON_VERSION
```

Comment:
This line is an exception for version 3.5, remove it (it seems now it works fine with this version)

Reply:
Aha. You mean removing the if or reverting the commit?

Reply:
It's better to remove the line :smile:

Reply:
Sorry for my late response ⏲️ 
Yeah, I meant to delete it 💯